packages/clerk-js/src/core/devBrowserHandler.ts

```diff
@@ -38,11 +38,14 @@ export default function createDevBrowserHandler({
 
   function setDevBrowserJWT(jwt: string) {
     localStorage.setItem(key, jwt);
+    // Append dev browser JWT to cookies, because server-side redirects (e.g. middleware) has no access to local storage
+    cookieHandler.setDevBrowserCookie(jwt);
```

Comment:
@anagstef Can we do this conditionally only for Next.js for now? We want to avoid sending the __client cookie in other frameworks as it confused customers in the past, especially for those who implement their own JWT verification mechanism.

Reply:
Discussed offline with Nikos.

Reply:
@SokratisVidros Unfortunately, we cannot distinguish which framework is used when executing the local interstitial. This is where this actually runs, when called by interstitial.